src/Order/Model/Item.php

```diff
@@ -137,6 +142,25 @@ public function setQuantity(int $quantity): Item
         return $this;
     }
 
+    /**
+     * @return string
+     */
+    public function getBillingText(): string
```

Comment:
the other getters return nullbable (in case it is not already set by a setter)

```suggestion
    public function getBillingText(): ?string
```